fr_CA/home/src/main/resources/rdf/display/firsttime/PropertyConfig_fr_CA.ttl

```diff
@@ -1,378 +1,380 @@
-# baseURI: http://vitroAnnotation_fr_CA

```

Comment:
Are there functional changes in this file? I cannot tell, given the large amount of whitespace change.